sale_commission_agent_restrict/__init__.py

```diff
@@ -0,0 +1,20 @@
+from . import models
+from odoo import api, SUPERUSER_ID
+
+# -- List of predefined rules that must be managed
+PREDEFINED_RULES = [
```

Comment:
It's better to add a hooks.py file at the module root in order to find it easily.